typescript/react/security/audit/react-http-leak.yaml

```diff
@@ -6,11 +6,13 @@ rules:
     message: >-
       this rule has been deprecated.
     metadata:
-      owasp: "A3: Sensitive Data Exposure"
+      owasp: 
+        - "A06:2007 - Information Leakage and Improper Error Handling"
+        - "A01:2021 - Broken Access Control"
       cwe: "CWE-200: Exposure of Sensitive Information to an Unauthorized Actor"
       deprecated: true
       references:
-        - https://github.com/cure53/HTTPLeaks
+        - http"s://github.com/cure53/HTTPLeaks
```

Comment:
i think this shouldn't have the extra double quotes

Reply:
Oh! Good eye! Fixed.